packages/shared/src/storage/AppleCloudKitStorage/AppleCloudKitStorage.ts

```
@@ -0,0 +1,69 @@
+  IAppleCloudKitAccountInfo,
+  IAppleCloudKitNativeModule,
+  IAppleCloudKitRecord,
+  IAppleCloudKitStorage,
+} from './types';
+
+export class AppleCloudKitStorage implements IAppleCloudKitStorage {
+  private getCloudKitModule(): IAppleCloudKitNativeModule {
+    if (platformEnv.isNativeIOS) {
+      const m = NativeModules?.CloudKitModule;
+      if (!m) {
+        throw new OneKeyLocalError('CloudKit native module not found');
```

ESLint

[error] Expected an error object to be thrown.

(@typescript-eslint/no-throw-literal)